src/coreclr/vm/jithelpers.cpp

```diff
@@ -5671,6 +5671,34 @@ HCIMPLEND_RAW
 EXTERN_C void JIT_ValidateIndirectCall();
 EXTERN_C void JIT_DispatchIndirectCall();
 
+HCIMPL2_VV(void*, JIT_DispatchVirtualStatic, MethodTable *constrainedType, MethodDesc *interfaceMethod)
```

Comment:
```suggestion
HCIMPL2(void*, JIT_DispatchVirtualStatic, MethodTable *constrainedType, MethodDesc *interfaceMethod)
```
`VV` is for methods that take arguments that do not fit into pointer size. Not the case here.